packages/jest-resolve/src/defaultResolver.ts

```diff
@@ -142,39 +142,50 @@ function resolveSync(
   }
 }
 
-/*
- * helper functions
- */
-function isFile(file: Config.Path): boolean {
-  let result;
+enum IPathType {
```

Comment:
It's not a big deal, but FYI-- not using the "prefix with I" convention in the Jest codebase.